package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "strapi-plugin-meilisearch",
-  "version": "0.6.0-strapi-v3",
+  "version": "0.6.0-strapi-v3.0",
```

Comment:
Does the file `playground/.tmp/data.db` should be here?

Reply:
I could not comment in the right place, but there is a file with that name and I'm not sure if it should be committed!